microservice-transaction-sample/rpc/build.gradle

```diff
@@ -20,16 +20,40 @@ protobuf {
     generateProtoTasks {
         all()*.plugins { grpc {} }
     }
-    generatedFilesBaseDir = "$projectDir/src"
 }
 
-archivesBaseName = "sample-rpc"
+base {
+    archivesName = "sample-rpc"
+}
+
+// Task copyGeneratedProtoToSrc copies the generated .java files into src directory
+task copyGeneratedProtoToSrc(type: Copy) {
+    description 'Copies generated Protocol Buffer classes to src/main/java/sample/rpc'
+    dependsOn generateProto
+    from "$buildDir/generated/source/proto/main/java/sample/rpc"
+    into "$projectDir/src/main/java/sample/rpc"
+    duplicatesStrategy = DuplicatesStrategy.INCLUDE
+}
+
+// Task deleteBuildMainJava deletes the generated .java files in build directory
+task deleteBuildMainJava(type: Delete) {
+    dependsOn copyGeneratedProtoToSrc
+    delete fileTree(dir: "$buildDir/generated/source/proto/main/java/sample/rpc")
+}
```

Comment:
Before running the compileJava task, the files under src/main/java must be deleted, otherwise we will get duplicate class errors like:
```
xxx/scalardb-samples/microservice-transaction-sample/rpc/build/generated/source/proto/main/java/sample/rpc/CommitRequest.java:9: エラー: クラスsample.rpc.CommitRequestが重複しています
public final class CommitRequest extends
```

Reply:
This task seems to delete only the generated protobuf files in the build directory, but the current task name might look to delete more. How about adding `GeneratedProtobuf` or something to the task name?

Reply:
@komamitsu  Thanks for reviewing! I applied your suggestion in [8427a0d](https://github.com/scalar-labs/scalardb-samples/pull/74/commits/8427a0ddbca87e9ba513e07c4bfe34436066e624). PTAL!